server/bin/pbench-audit-server.sh

```diff
@@ -201,11 +201,14 @@ function verify_incoming {
         tarball_dirs="${workdir}/tarballdirs"
         empty_tarball_dirs="${workdir}/emptytarballdirs"
         > ${empty_tarball_dirs}.unsorted
+        unpacking_tarball_dirs="${workdir}/unpackingtarballdirs"
+        > ${unpacking_tarball_dirs}
```

Comment:
Shouldn't  the `tarball_dirs` and `tarball_links` files be initialized to empty as well?

Reply:
One can do that, but `-fprintf` of `find` either creates, or overwrites the file given.

Reply:
But then why do it for two, and not for the other two? Let `-fprintf` handle it for all of them or (redundantly I realize) initialize all of them. It's the inconsistency that bugs me.

Reply:
Fixed.